app/scripts/directives/fromFile.js

```diff
@@ -118,6 +118,7 @@ angular.module("openshiftConsole")
             return;
           }
 
+          $scope.resource = resource;
```

Comment:
Shouldn't need to set this as a scope var, can just use `resource` below when you call `createFromTemplateURL` below